### DISCUSSION TITLE
React-MarkDown Changelog

### DISCUSSION THREAD

**Discussion by crs1138** [remarkjs/remark#1167] [Q&A] [answered]
Is there a changelog for react-markdown? I gotta downgrade to version 6 as I can't get the package to play nicely with Jest and I want to know, what functionality I'll be losing and what sort of security implications this might have. Our environment is secure (= no outer world MD input).

**Comment by wooorm** [comment, marked as answer]
Most projects on GitHub, I find, have changelogs in the Releases tab shown on GitHub: https://github.com/remarkjs/react-markdown/releases.
Some also have them as files, in this case the `changelog.md` file in the `react-markdown` repo: https://github.com/remarkjs/react-markdown/blob/main/changelog.md
